config/locales/en/documents/show.yml

```diff
@@ -32,6 +32,7 @@ en:
         status: Status
         updated_at: Last updated
         created_at: Created
+        created_by: Document created by
```

Comment:
We could pass in the name as a variable as per https://github.com/alphagov/content-publisher/blob/b62190d6873105e2cc244dc5e502e980aaa724b7/config/locales/en/validations.yml#L5 and then you wouldn't need to concatenate the string after?